README.md

```diff
@@ -20,16 +20,31 @@ Learn the fundamentals of web development with our 12-week comprehensive course
 
 Visit [**Student Hub page**](https://docs.microsoft.com/learn/student-hub/?WT.mc_id=academic-77807-sagibbon) where you will find beginner resources, Student packs and even ways to get a free certificate voucher. This is the page you want to bookmark and check from time to time as we switch out content monthly.
 
-### 📣 Announcement - _New Curriculum_ on Generative AI for JavaScript was just released!
+### 📣 Announcement - _New Curriculum_ on Generative AI for JavaScript was just released
 
 Don't miss our NEW 5-lesson curriculum on generative AI!
```

Comment:
I'd keep this really simple since we'll add more chapters. Maybe something like:

Don't miss our new Generative AI course!